src/SeqViz/handlers/selection.tsx

```diff
@@ -34,21 +35,40 @@ export const defaultSelection = {
 export const SelectionContext = React.createContext(defaultSelection);
 SelectionContext.displayName = "SelectionContext";
 
+export interface SeqVizMouseEventType {
```

Comment:
This looks good but can we use the normal synthetic mouse event type (rather than redefine)?

Eg here: https://fettblog.eu/typescript-react/events/